internal/sdkv2provider/resource_cloudflare_workers_script.go

```diff
@@ -131,6 +131,14 @@ func parseWorkerBindings(d *schema.ResourceData, bindings ScriptBindings) {
 	}
 }
 
+func getCompatibilityFlags(d *schema.ResourceData) []string {
+	compatibilityFlags := make([]string, 0)
+	for _, item := range d.Get("compatibility_flags").([]interface{}) {
```

Comment:
when you swap to a `TypeSet` this will become
```suggestion
	for _, item := range d.Get("compatibility_flags").(*schema.Set).List() {
```